<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>BasicSettingHomeUi</class>
 <widget class="QWidget" name="BasicSettingHomeUi">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1060</width>
    <height>720</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>0</width>
    <height>0</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QPushButton#pb_acceleration, #pb_magnetic_field, #pb_perspective, #pb_coef, #pb_restore_coef {
background-color: #007AFF;
border-radius: 4px;
font-family: SourceHanSansCN-Normal;
font-size: 14px;
color: #FFFFFF;
text-align: center;
}

QLabel#lb_show_coef, #lb_show_pic {
background-color: transparent;
font-family: SourceHanSansCN-Normal;
font-size: 12px;
color: #0B22C6;
text-align: center;
}</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_2" stretch="0">
   <property name="spacing">
    <number>0</number>
   </property>
   <property name="leftMargin">
    <number>20</number>
   </property>
   <property name="topMargin">
    <number>10</number>
   </property>
   <property name="rightMargin">
    <number>20</number>
   </property>
   <property name="bottomMargin">
    <number>30</number>
   </property>
   <item>
    <widget class="QFrame" name="frame">
     <property name="styleSheet">
      <string notr="true">.QFrame {
border-radius: 10px;
background-color: #FFFFFF;
border: 1px solid #E0E0E0;
}
</string>
     </property>
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout">
      <property name="spacing">
       <number>14</number>
      </property>
      <property name="leftMargin">
       <number>0</number>
      </property>
      <property name="topMargin">
       <number>110</number>
      </property>
      <property name="rightMargin">
       <number>0</number>
      </property>
      <property name="bottomMargin">
       <number>0</number>
      </property>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout">
        <property name="leftMargin">
         <number>408</number>
        </property>
        <item alignment="Qt::AlignLeft">
         <widget class="QLabel" name="label">
          <property name="maximumSize">
           <size>
            <width>204</width>
            <height>134</height>
           </size>
          </property>
          <property name="text">
           <string/>
          </property>
          <property name="pixmap">
           <pixmap resource="../resource/resource.qrc">:/ui/images/img_ucar02.png</pixmap>
          </property>
          <property name="scaledContents">
           <bool>true</bool>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_2" stretch="0,0,0,0">
        <property name="spacing">
         <number>30</number>
        </property>
        <property name="leftMargin">
         <number>235</number>
        </property>
        <property name="topMargin">
         <number>10</number>
        </property>
        <property name="rightMargin">
         <number>0</number>
        </property>
        <item alignment="Qt::AlignLeft">
         <widget class="QLabel" name="label_2">
          <property name="minimumSize">
           <size>
            <width>105</width>
            <height>0</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>105</width>
            <height>16777215</height>
           </size>
          </property>
          <property name="styleSheet">
           <string notr="true">QLabel {
font-family: SourceHanSansCN-Medium;
font-size: 18px;
color: #333333;
text-align: right;
}</string>
          </property>
          <property name="text">
           <string>校准：</string>
          </property>
         </widget>
        </item>
        <item alignment="Qt::AlignLeft">
         <widget class="QPushButton" name="pb_acceleration">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="minimumSize">
           <size>
            <width>140</width>
            <height>40</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>140</width>
            <height>40</height>
           </size>
          </property>
          <property name="text">
           <string>加速度</string>
          </property>
          <property name="flat">
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item alignment="Qt::AlignLeft">
         <widget class="QPushButton" name="pb_magnetic_field">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="minimumSize">
           <size>
            <width>140</width>
            <height>40</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>140</width>
            <height>40</height>
           </size>
          </property>
          <property name="text">
           <string>磁场</string>
          </property>
          <property name="flat">
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item>
         <spacer name="horizontalSpacer_5">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_3">
        <property name="spacing">
         <number>10</number>
        </property>
        <property name="leftMargin">
         <number>385</number>
        </property>
        <item>
         <widget class="QLabel" name="label_4">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="minimumSize">
           <size>
            <width>20</width>
            <height>20</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>20</width>
            <height>20</height>
           </size>
          </property>
          <property name="text">
           <string/>
          </property>
          <property name="pixmap">
           <pixmap resource="../resource/resource.qrc">:/ui/images/ic_default_option.png</pixmap>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="label_3">
          <property name="styleSheet">
           <string notr="true">QLabel {
font-family: SourceHanSansCN-Normal;
font-size: 12px;
color: #333333;
text-align: center;
}</string>
          </property>
          <property name="text">
           <string>陀螺仪自动校准</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_4" stretch="0,0,0,0,0">
        <property name="spacing">
         <number>30</number>
        </property>
        <property name="leftMargin">
         <number>235</number>
        </property>
        <property name="topMargin">
         <number>20</number>
        </property>
        <property name="rightMargin">
         <number>0</number>
        </property>
        <item alignment="Qt::AlignLeft">
         <widget class="QLabel" name="label_5">
          <property name="minimumSize">
           <size>
            <width>105</width>
            <height>0</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>105</width>
            <height>16777215</height>
           </size>
          </property>
          <property name="styleSheet">
           <string notr="true">QLabel {
font-family: SourceHanSansCN-Medium;
font-size: 18px;
color: #333333;
text-align: right;
}</string>
          </property>
          <property name="text">
           <string>标定：</string>
          </property>
         </widget>
        </item>
        <item alignment="Qt::AlignLeft">
         <widget class="QPushButton" name="pb_perspective">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="minimumSize">
           <size>
            <width>140</width>
            <height>40</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>140</width>
            <height>40</height>
           </size>
          </property>
          <property name="styleSheet">
           <string notr="true">QPushButton {
background-color: #007AFF;
border-radius: 4px;
font-family: SourceHanSansCN-Normal;
font-size: 14px;
color: #FFFFFF;
text-align: center;
}</string>
          </property>
          <property name="text">
           <string>摄像头标定</string>
          </property>
          <property name="flat">
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="pb_coef">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="minimumSize">
           <size>
            <width>140</width>
            <height>40</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>140</width>
            <height>40</height>
           </size>
          </property>
          <property name="styleSheet">
           <string notr="true">QPushButton {
background-color: #007AFF;
border-radius: 4px;
font-family: SourceHanSansCN-Normal;
font-size: 14px;
color: #FFFFFF;
text-align: center;
}</string>
          </property>
          <property name="text">
           <string>内参</string>
          </property>
          <property name="flat">
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="pb_restore_coef">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="minimumSize">
           <size>
            <width>230</width>
            <height>40</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>230</width>
            <height>40</height>
           </size>
          </property>
          <property name="styleSheet">
           <string notr="true">QPushButton {
background-color: #007AFF;
border-radius: 4px;
font-family: SourceHanSansCN-Normal;
font-size: 14px;
color: #FFFFFF;
text-align: center;
}</string>
          </property>
          <property name="text">
           <string>恢复出厂内参</string>
          </property>
          <property name="flat">
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item>
         <spacer name="horizontalSpacer_6">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_5">
        <property name="spacing">
         <number>58</number>
        </property>
        <property name="leftMargin">
         <number>489</number>
        </property>
        <item>
         <widget class="QLabel" name="lb_show_coef">
          <property name="text">
           <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;a href=&quot;http://&quot;&gt;&lt;span style=&quot; text-decoration: underline; color:#0000ff;&quot;&gt;查看内参参数&lt;/span&gt;&lt;/a&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
          </property>
          <property name="textFormat">
           <enum>Qt::RichText</enum>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="lb_show_pic">
          <property name="text">
           <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;a href=&quot;http://&quot;&gt;&lt;span style=&quot; text-decoration: underline; color:#0000ff;&quot;&gt;查看内参标定照片&lt;/span&gt;&lt;/a&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
          </property>
          <property name="textFormat">
           <enum>Qt::RichText</enum>
          </property>
         </widget>
        </item>
        <item>
         <spacer name="horizontalSpacer_7">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_6" stretch="0,0,0">
        <property name="spacing">
         <number>30</number>
        </property>
        <property name="leftMargin">
         <number>235</number>
        </property>
        <property name="topMargin">
         <number>20</number>
        </property>
        <property name="rightMargin">
         <number>0</number>
        </property>
        <item alignment="Qt::AlignLeft">
         <widget class="QLabel" name="label_6">
          <property name="minimumSize">
           <size>
            <width>105</width>
            <height>0</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>105</width>
            <height>16777215</height>
           </size>
          </property>
          <property name="styleSheet">
           <string notr="true">QLabel {
font-family: SourceHanSansCN-Medium;
font-size: 18px;
color: #333333;
text-align: right;
}</string>
          </property>
          <property name="text">
           <string>其他：</string>
          </property>
         </widget>
        </item>
        <item alignment="Qt::AlignLeft">
         <widget class="QPushButton" name="pb_rollback">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="minimumSize">
           <size>
            <width>240</width>
            <height>40</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>240</width>
            <height>40</height>
           </size>
          </property>
          <property name="styleSheet">
           <string notr="true">QPushButton {
background-color: #007AFF;
border-radius: 4px;
font-family: SourceHanSansCN-Normal;
font-size: 14px;
color: #FFFFFF;
text-align: center;
}</string>
          </property>
          <property name="text">
           <string>前轮回中校准</string>
          </property>
          <property name="flat">
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item>
         <spacer name="horizontalSpacer_8">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
      </item>
      <item>
       <spacer name="verticalSpacer">
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>20</width>
          <height>40</height>
         </size>
        </property>
       </spacer>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="../resource/resource.qrc"/>
 </resources>
 <connections/>
</ui>
